extensions/wikia/PortableInfobox/styles/PortableInfobox.scss

```diff
@@ -274,5 +273,10 @@ $infobox-debug-error-msg-bg: black;
 			margin: 0;
 		}
 	}
+
+	.portable-infobox-item-value {
+		font-size: 12px;
+		line-height: 18px;
+	}
```

Comment:
have you checked if tabular layout data-value fields are aligned ? () they were for `line-height: 19px;`